client/src/App.jsx

```diff
@@ -30,6 +31,7 @@ const App = () => {
           />
         </Suspense>
       </Switch>
+      <ReactQueryDevtools initialIsOpen={false} />
```

Comment:
What is this for?

Reply:
this is dev tools from a library called react query to checko whether i am getting data properly or not, avoid console long. dont worry it wont pop up in production

Reply:
Ok

Reply:
easy to track what data i am fetching thats why use it

![Screenshot 2020-10-29 160659](https://user-images.githubusercontent.com/35744979/97626818-cdccb800-1a00-11eb-99bd-f0858b4bdcf2.png)